.storybook/preview-head.html

```diff
@@ -0,0 +1,49 @@
+<style type="text/css">
+  @font-face {
+    font-family: 'Karu';
+    src: url('fonts/Karu-Regular.woff') format('woff');
+    font-weight: 400;
+    font-style: normal;
+  }
+
+  @font-face {
+    font-family: 'Karu';
+    src: url('fonts/Karu-Light.woff') format('woff');
+    font-weight: 300;
+    font-style: normal;
+  }
+
+  @font-face {
+    font-family: 'Karu';
+    src: url('fonts/Karu-Medium.woff') format('woff');
+    font-weight: 500;
+    font-style: normal;
+  }
+
+  @font-face {
+    font-family: 'Karu';
+    src: url('fonts/Karu-Bold.woff') format('woff');
+    font-weight: 700;
+    font-style: normal;
+  }
+
+  @font-face {
+    font-family: 'Karu';
+    src: url('fonts/Karu-ExtraBold.woff') format('woff');
+    font-weight: 800;
+    font-style: normal;
+  }
+
+  h1,
+  h2,
+  h3,
+  h4,
+  a,
+  p {
+    font-family: 'Karu', sans-serif;
+  }
+
+  html {
+    font-size: 10px;
```

Comment:
Do we need this here?

Reply:
This will come in handy when we start writing the media queries because we are using the `rem` unit, so we can just change the HTML font size in the media query and things are going to change dynamically

Reply:
Ah yeh, makes sense!